src/scripts/lychee.js

```diff
@@ -418,6 +424,12 @@ lychee.html = function(literalSections, ...substs) {
 
 lychee.error = function(errorThrown, params, data) {
 
+	// in this case, our cookie has probably expired
+	if (data.startsWith('Error: Function not available for guests.')) {
```

Comment:
Should use `data===` instead. startsWith is not supported by IE11. Not worth to break support for this function.